src/otx/core/data/dataset/keypoint_detection.py

```diff
@@ -55,15 +55,14 @@ def __init__(
 
         self.dm_subset = self._get_single_bbox_dataset(dm_subset)
 
-        if self.dm_subset.categories():
+        # arrow doesn't follow common coco convention, no need to fetch kp-specific labels
+        if self.dm_subset.categories() and data_format != "arrow":
```

Comment:
What would be set for self.label_info if this condition is not fulfilled?

Reply:
In that case, label info generated by parent class is used. For kp arrow common labels extraction procedure works correctly, while for kp filesystem coco dataset is doesn't work, and we need another one here